mne/channels/_standard_montage_utils.py

```diff
@@ -294,7 +294,7 @@ def _read_elp_besa(fname, head_size):
     # This .elp is not the same as polhemus elp. see _read_isotrak_elp_points
     dtype = np.dtype('S8, S8, f8, f8, f8')
     try:
-        data = np.loadtxt(fname, dtype=dtype, skip_header=1)
+        data = np.loadtxt(fname, dtype=dtype, skiprows=1)
```

Comment:
as you can see now the code in the try and except are the same with this fix. I have no idea why we have this skip_header which I don't see in numpy documentation. Maybe an old version. Can you simply remove the try and see if CIs are happy? 🙏

Reply:
We can get rid of the try/except entirely, our min is now 1.18.1 which already has the `skiprows` argument

https://numpy.org/doc/1.18/reference/generated/numpy.loadtxt.html